lib/maths/time_series/CTimeSeriesModel.cc

```diff
@@ -273,6 +274,20 @@ class CTimeSeriesAnomalyModel {
     //! Persist by passing information to \p inserter.
     void acceptPersistInserter(core::CStatePersistInserter& inserter) const;
 
+    double sumPredictionError() const {
+        if (!m_Anomaly) {
```

Comment:
nit (conventions)
```suggestion
        if (m_Anomaly == std::nullopt) {
```
And same below.